packages/core/contracts/financial-templates/common/financial-product-libraries/KpiOptionsFinancialProductLibrary.sol

```diff
@@ -0,0 +1,26 @@
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+import "./FinancialProductLibrary.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "../../../common/implementation/Lockable.sol";
+
+/**
+ * @title KPI Options Financial Product Library
+ * @notice Returns a transformed collateral requirement. The collateralization requirement should
+ * always be equivalent to being collateralized by 2 tokens, no matter what the price or expiration status is.
+ */
+contract KpiOptionsFinancialProductLibrary is FinancialProductLibrary, Ownable, Lockable {
+    /**
+     * @notice Returns a transformed collateral requirement that is always set to 2 tokens.
+     * @param oraclePrice price from the oracle to transform the collateral requirement.
+     * @param collateralRequirement financial products collateral requirement to be scaled to a flat rate.
+     * @return transformedCollateralRequirement the input collateral requirement with the transformation logic applied to it.
+     */
+    function transformCollateralRequirement(
```

Comment:
Should we also be implementing `transformPrice` which returns `2` is the request time is before the expiration time? @mrice32 @chrismaree

Reply:
Yeah I think you're right. It doesn't make much sense as is right now without any price transformation. Setting a CR of 2 isn't actually a constant token amount, unless we also make the price constant pre-expiry.

Reply:
I'm thinking about this some more, and I think `transformPrice` might actually have to return `1` pre-expiry. The liquidation calculation pre-expiry is as follows:

```
liquidation.settlementPrice = transformPrice(liquidationTime);
FixedPoint.Unsigned memory tokenRedemptionValue =
            liquidation.tokensOutstanding.mul(liquidation.settlementPrice);
FixedPoint.Unsigned memory requiredCollateral =
            tokenRedemptionValue.mul(transformCollateralRequirement(liquidation.settlementPrice));
disputeSucceeded = liquidation.liquidatedCollateral.isGreaterThanOrEqual(requiredCollateral);
```

So if `transformCollateralRequirement` always returns `2` and `transformPrice` always returns 1, then as long as the contract is seeded with at least 2x collateral tokens than tokens minted, then the contract will always be correctly collateralized pre-expiry (assuming no withdrawals are allowed).

Reply:
Ok disregard because I see the `transformCollateralRequirement` always returns 1 now

Reply:
Sorry, my outdated comment was probably confusing. The collateral requirement transformation was updated to always return 1.